rules/no-single-promise-in-promise-methods.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+const isPromiseMethodWithArray = require('./utils/is-promise-method-with-array.js');
+
+const MESSAGE_ID = 'no-single-promise-in-promise-methods';
+const messages = {
+	[MESSAGE_ID]: 'Parameter in `Promise.{{method}}` should not be a single element array.',
+};
+
+const isPromiseMethodWithSinglePromise = node =>
+	isPromiseMethodWithArray(node)
+	&& node.arguments[0].elements.length === 1;
+
+const getMethodName = node => node.callee.property.name;
+
+const getFixer = ({sourceCode}, node) => fixer =>
+	fixer.replaceText(node, sourceCode.getText(node.arguments[0].elements[0]));
```

Comment:
This will break

```js
await Promise.all([(a, b)])
```

Reply:
It's corrected.

Reply:
I don't know how you fix it, but it's incorrect. Please try not to force push, I can't see changes.

Reply:
The fix was to limit the types by excluding SequenceExpression.
Sorry about the push force.